tests/Unit/NumericalAlgorithms/Interpolation/Test_MultiLinearSpanInterpolation.cpp

```diff
@@ -125,24 +127,28 @@ void test() {
     general_intp.extrapolate_below_data(d, true);
   }
 
-  auto y_expected = f(x);
-
+  auto y_expected = mock_function(point);
 
-  auto y_interpolated = uniform_intp.interpolate(which_var, x);
-  auto y_interpolated_gen = general_intp.interpolate(which_var, x);
+  auto y_interpolated = uniform_intp.interpolate(which_var, point);
+  auto y_interpolated_gen = general_intp.interpolate(which_var, point);
 
+  const double epsilon = 1.0e-12;
+  CAPTURE(epsilon);
   for (size_t nv = 0; nv < NumVar; ++nv) {
+    CAPTURE(y_expected[nv]);
```

Comment:
looks like clang-tidy wants these using gsl::at too (or some subset of them at least)

Reply:
Oops, my bad. Squashed those in as well.